extension/chrome/elements/attachment.ts

```diff
@@ -83,15 +103,19 @@ export class AttachmentDownloadView extends View {
     this.renderHeader();
     $('#name').attr('title', this.name || '');
     $('img#file-format').attr('src', this.getFileIconSrc());
-    if (!this.size && this.url) { // download url of a file that has an unknown size
-      this.getUrlFileSize(this.url!).then(fileSize => {
-        if (typeof fileSize !== 'undefined') {
-          this.size = fileSize;
-        }
-      }).catch(ApiErr.reportIfSignificant);
+    if (!this.size && this.url) {
+      // download url of a file that has an unknown size
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      this.getUrlFileSize(this.url!)
+        .then(fileSize => {
+          if (typeof fileSize !== 'undefined') {
+            this.size = fileSize;
+          }
+        })
+        .catch(ApiErr.reportIfSignificant);
     }
     try {
-      this.canClickOnAttachment = ! await this.processAsPublicKeyAndHideAttachmentIfAppropriate();
+      this.canClickOnAttachment = !(await this.processAsPublicKeyAndHideAttachmentIfAppropriate());
```

Comment:
I suppose this is driven by some new eslint rule? `! await something;` is intuitive and nice to me, but maybe because I'm used to it. How do you see this?

Reply:
I used to `!(await ...)` and it's default behavior of `prettier` formatter, probably because of consistency for different cases like:
```js
if (!(await wkd.lookupEmail(email)).pubkeys.length) { ...

// because it won't work if you'll forget to add brackets and write it this way:
if (! await wkd.lookupEmail(email).pubkeys.length) {...
```

Reply:
Yup, in the second case the brackets need to be added.

Doesn't bother me that much. I just miss the clean look, since there are very few examples of the second case, I think.